ui/imports/shared/views/profile/ProfileShowcaseView.qml

```diff
@@ -124,385 +129,392 @@ Control {
-                    id: communitiesView
-                    width: 606
-                    height: parent.height
-                    anchors.top: parent.top
-                    anchors.topMargin: Style.current.halfPadding
-                    anchors.bottom: parent.bottom
-                    anchors.bottomMargin: Style.current.halfPadding
-                    anchors.horizontalCenter: parent.horizontalCenter
-                    anchors.horizontalCenterOffset: Style.current.halfPadding
-                    clip: false
-                    cellWidth: d.delegateWidthM
-                    cellHeight: d.delegateHeightM
-                    visible: count
-                    model: communitiesProxyModel
-                    ScrollBar.vertical: StatusScrollBar { }
-                    delegate: StatusCommunityCard {
-                        id: profileDialogCommunityCard
-                        readonly property var permissionsList: model.permissionsModel //TODO: Add permissions model in the community model
-                        readonly property bool requirementsMet: !!model.allTokenRequirementsMet ? model.allTokenRequirementsMet : false
-                        cardSize: StatusCommunityCard.Size.Small
-                        width: GridView.view.cellWidth - Style.current.padding
-                        height: GridView.view.cellHeight - Style.current.padding
-                        titleFontSize: 15
-                        descriptionFontSize: 12
-                        communityId: model.id ?? ""
-                        loaded: !!model.id
-                        asset.source: model.image ?? ""
-                        asset.isImage: !!model.image
-                        asset.width: 32
-                        asset.height: 32
-                        name: model.name ?? ""
-                        memberCountVisible: false
-                        layer.enabled: hovered
-                        border.width: hovered ? 0 : 1
-                        border.color: Theme.palette.baseColor2
-                        descriptionFontColor: Theme.palette.baseColor1
-                        description: {
-                            switch (model.memberRole)  {
-                            case (Constants.memberRole.owner):
-                                return qsTr("Owner");
-                            case (Constants.memberRole.admin) :
-                                return qsTr("Admin");
-                            case (Constants.memberRole.tokenMaster):
-                                return qsTr("Token Master");
-                            default:
-                                return qsTr("Member");
-                            }
+            StatusGridView {
+                id: communitiesView
+                width: 606
+                height: parent.height
+                anchors.top: parent.top
+                anchors.topMargin: (webView.contentY > Style.current.halfPadding) ? 1 : Style.current.bigPadding
```

Comment:
Why do we need here a cross reference to `webView`?? :thinking: 
Same question for all tab items